### ISSUE TITLE
Terraform 0.12 compatibility

### ISSUE THREAD

**Issue by alias-dev** [nutanix/terraform-provider-nutanix#66]
**Describe the bug**
Running terraform init with terraform >= 0.12 throws a compatibility error

**Expected behavior**
Successful initialization

**Logs**
```
Initializing provider plugins...
- Checking for available provider plugins...

No available provider "nutanix" plugins are compatible with this Terraform version.

From time to time, new Terraform major releases can change the requirements for
plugins such that older plugins become incompatible.

Terraform checked all of the plugin versions matching the given constraint:
    1.0.1

Unfortunately, none of the suitable versions are compatible with this version
of Terraform. If you have recently upgraded Terraform, it may be necessary to
move to a newer major release of this provider. Alternatively, if you are
attempting to upgrade the provider to a new major version you may need to
also upgrade Terraform to support the new version.

Consult the documentation for this provider for more information on
compatibility between provider versions and Terraform versions.


Error: no available version is compatible with this version of Terraform
```

**Versions:**
 - OS: macOS Mojave 10.14.5
 - Terraform: v0.12.3
 - Nutanix Cluster: 5.10.5
 - Nutanix Prism Central: 5.10.5
 - Terraform provider: v1.0.1

**Comment by csawyerYumaed** [comment]
same here. It's unhappy with 0.12.3

downloading 0.11.14 works:

Terraform v0.11.14
+ provider.nutanix v1.0.1

**Comment by marinsalinas** [comment]
We are working on it, I hope to get the new provider version next week.

Thanks!

**Comment by marinsalinas** [comment]
Closed by #72
